src/main/java/core/basesyntax/reportcreator/ReportCreator.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.reportcreator;
+
+import core.basesyntax.Operation;
+import core.basesyntax.Storage;
+import java.util.Arrays;
+import java.util.Map;
+
+public class ReportCreator {
+    public String generateReport(Storage storage) {
+        StringBuilder report = new StringBuilder("type,fruit,quantity\n");
+        for (Map.Entry<String, Integer> entry : Storage.fruits.entrySet()) {
+            report.append(Arrays.stream(Operation.values())
+                            .map(Operation::toString).findFirst().get())
+                    .append(",")
+                    .append(entry.getKey())
+                    .append(",")
+                    .append(entry.getValue())
+                    .append("\n"
+                    );
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)